stacks-common/src/util/log.rs

```diff
@@ -41,15 +43,24 @@ fn print_msg_header(mut rd: &mut dyn RecordDecorator, record: &Record) -> io::Re
     write!(rd, " ")?;
 
     rd.start_timestamp()?;
-    let elapsed = SystemTime::now()
-        .duration_since(SystemTime::UNIX_EPOCH)
-        .unwrap_or(Duration::from_secs(0));
-    write!(
-        rd,
-        "[{:5}.{:06}]",
-        elapsed.as_secs(),
-        elapsed.subsec_nanos() / 1000
-    )?;
+    let system_time = SystemTime::now();
+    match &*STACKS_LOG_FORMAT_TIME {
```

Comment:
I think semantically it would make more sense if this was `None` vs `Some` rather than using the error.

From the perspective of the program, it's not really an error, so it's semantically confusing.

Reply:
Makes sense. env::var returns a Result so I'll use ok() to convert into Option.